etcdserver/server.go

```diff
@@ -869,6 +875,66 @@ func (s *EtcdServer) LeaseHandler() http.Handler {
 
 func (s *EtcdServer) RaftHandler() http.Handler { return s.r.transport.Handler() }
 
+type ServerPeerHTTP interface {
+	ServerPeer
+	ServerDowngradeHTTP
+}
+
+type ServerDowngradeHTTP interface {
+	// DowngradeInfo is the downgrade information of the cluster
+	DowngradeInfo() *membership.DowngradeInfo
```

Comment:
Is this API needed to be packed in this interface?

Reply:
Removed `DowngradeInfo()`